modules/jupyter-widget/src/create-deck.js

```diff
@@ -10,39 +8,25 @@ import {Tile3DLoader} from '@loaders.gl/3d-tiles';
 import {LASWorkerLoader} from '@loaders.gl/las';
 import * as loaders from '@loaders.gl/core';
 
-import {
-  COORDINATE_SYSTEM,
-  Deck,
-  MapView,
-  FirstPersonView,
-  OrbitView,
-  OrthographicView,
-  log
-} from '@deck.gl/core';
-
-// import {DeckGL} from '@deck.gl/core/bundle';
-import * as Layers from '@deck.gl/layers';
-import * as AggregationLayers from '@deck.gl/aggregation-layers';
-import * as GeoLayers from '@deck.gl/geo-layers';
-import * as MeshLayers from '@deck.gl/mesh-layers';
+import * as deck from './deck-bundle';
+
 import GL from '@luma.gl/constants';
 
-import {JSONConverter} from '@deck.gl/json';
+function extractClasses() {
+  // Get classes for registration from standalone deck.gl
+  const classesDict = {};
+  const classes = Object.keys(deck).filter(x => x.charAt(0) === x.charAt(0).toUpperCase());
+  classes.map(k => (classesDict[k] = deck[k]));
+  return deck;
+}
 
 function createDeckWithImports(args) {
```

Comment:
- Rename this function - it no longer imports/requires anything
- Use JSDoc to document the arguments expected